examples/run_CPMG.py

```diff
@@ -12,21 +12,21 @@
 carrierFreq_MHz = 14.895548
```

Comment:
we don't have a ppg for CPMG just yet so going forward this will need to be a new PR. I have only adjusted names of parameters to include units and the saving style of the acquisition parameters